lib/install/coffee.rb

```diff
@@ -12,6 +12,9 @@
   "environment.loaders.append('coffee', coffee)\n",
   before: "module.exports"
 
+say "Updating webpack paths to include .coffee file extension"
+insert_into_file Webpacker.config.config_path, "    - .coffee\n", after: /extensions:\n/
```

Comment:
Inserting YAML like this at the corrected indentation level seems a little brittle. Do we do it this way elsewhere in Rails?

Reply:
Couldn't find any reference but this is how it's done in Rails: https://github.com/rails/rails/blob/63f0c04850dd0bcdc7d35266e81fa1a7778570a8/railties/lib/rails/generators/actions.rb#L326

May be we can borrow that method.